swagger/widgets/widgets_and_boards.json

```diff
@@ -0,0 +1,596 @@
+          "uid": {
+            "type": "string",
+            "description": "The widget uid"
+          },
+          "dimentions": {
+            "description": "The sections widgets",
+            "allOf": [
+              {
+                "$ref": "#/components/schemas/CustomizedWidgetDimentionsDto"
+              }
+            ]
+          }
+        },
+        "required": [
+          "uid"
+        ]
+      }
+    }
+  },
+  "paths": {
+    "/v3/widgets": {
+      "post": {
+        "operationId": "Widgets_create",
+        "summary": "Create a Widget",
+        "parameters": [],
+        "requestBody": {
+          "required": true,
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/WidgetRequestDto"
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    {
+                      "$ref": "#/components/schemas/Response"
+                    },
+                    {
+                      "properties": {
+                        "data": {
+                          "type": "object",
+                          "$ref": "https://vcita.github.io/developers-hub/entities/dashboard/widget.json"
+                        }
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "tags": [
+          "Widgets"
+        ],
+        "security": [
+          {
+            "bearer": []
+          }
+        ]
+      },
+      "get": {
+        "operationId": "Widgets_getAll",
+        "summary": "Get all Widgets",
+        "parameters": [
+          {
+            "name": "page",
+            "required": false,
+            "in": "query",
+            "description": "Page number of results.",
+            "schema": {
+              "minimum": 1,
+              "maximum": 10000,
+              "default": 1,
+              "type": "number"
+            }
+          },
+          {
+            "name": "per_page",
+            "required": false,
+            "in": "query",
+            "description": "How many items to return per page. Max: 100",
+            "schema": {
+              "minimum": 1,
+              "maximum": 100,
+              "default": 25,
+              "type": "number"
+            }
+          },
+          {
+            "description": "Sort by fields. Example: sort=created_at:asc,updated_at:desc",
+            "name": "sort",
+            "required": false,
+            "in": "query",
+            "schema": {
+              "default": "updated_at:desc",
+              "type": "string"
+            }
+          },
+          {
+            "description": "Filter by a field. Example: filter[created_at][gt]=2019-01-01",
```

Comment:
We do not support this filtering anymore, please remove it. If you need a filter, please declare it explicitly